.changelog/2687.feature.2.md

```diff
@@ -0,0 +1,16 @@
+`oasis-node`: Add benchmarking utilities
+
+- Renames `--metrics.push.job_name` to `--metrics.job_name` and
```

Comment:
I think the metrics config changes should be under a breaking segment (as existing node operators will need to update configs)